src/systemTest/java/com/flightstats/hub/testcase/CallbackServerHelper.java

```diff
@@ -34,7 +34,7 @@
     private CallbackResource callbackResource;
```

Comment:
system test directory structure has changed. These classes no longer exist. I will try to get the structure in tot he master so that we do not have to deal with outdated classes.